pkg/generate/app/sourcelookup.go

```diff
@@ -90,6 +90,7 @@ type SourceRepository struct {
 	buildWithDocker  bool
 	ignoreRepository bool
 	binary           bool
+	dockerfile       bool
```

Comment:
this needs to be more specific... does this mean "extract the dockerfile from the repo?"